packages/factory/src/helpers.ts

```diff
@@ -119,8 +126,10 @@ export const getContext = ({
         build,
         // This will be updated in the bundler-report plugin once we have the configuration.
         cwd,
-        inject: (item: ToInjectItem) => {
-            injections.set(getUniqueId(), item);
+        getLogger: getLoggerFactory(build, options.logLevel),
+        // This will be updated in the injection plugin on initialization.
+        inject: () => {
+            throw new Error('Inject function called before it was initialized.');
```

Comment:
❓**question** I'm not very familiar with the code but is that error caught and reported properly elsewhere or could it just break the build?